_component_design/left-nav.md

```diff
@@ -4,135 +4,168 @@ parent: Navigation
 layout: component
 category: Components
 usage: >
-  Left navigation establishes a user's location within an application and shows the path forward to accomplish a goal. Consistent navigation patterns increase user confidence and efficiency.
+  The left navigation provides a menu of navigation items that a user clicks to move to another page or location within the application they are using. Because it is easier for user to scan, left navigation is preferred over top navigation. Consistent navigation patterns increase user confidence and efficiency.
 preview-image: preview-images/side-nav.svg
 status: stable
 resource: true
 helix-ui-css: true
 helix-ui-javascript: true
-last-modified: 2017-12-11
+last-modified: 2018-11-05
 pagelink: https://rackerlabs.github.io/helix-ui/components/navigation/
 ---
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Unless a firm implementation constraint requires top navigation, use the left navigation component for web applications.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### When to use
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-Use the left navigation component for web applications, unless there is a firm implementation constraint that requires top navigation.
+## Best practices
 
-### Best practices
+Consider the following best practices when you construct the left navigation:
 
-- The preferred implementation includes only primary and secondary navigation links.
-- An alternative, non-preferred implementation includes [tertiary](#tertiary-left-navigation) navigation links.
+- Implement only primary and secondary navigation items.
+- If you have a very large information architecture, you can use tertiary navigation items. However, in most cases we recommend that you compress or sort the information architecture so that only primary and secondary links are needed.
```

Comment:
s/we recommend that you compress or sort the information architecture so that only primary and secondary links are needed./ we recommend that you refactor the information architecture so that only two levels (primary and secondary) are needed.

Reply:
Done.